pkg/webhooks/namespace/namespace.go

```diff
@@ -123,39 +168,40 @@ func (s *NamespaceWebhook) Authorized(request admissionctl.Request) admissionctl
 // Is the request authorized?
 func (s *NamespaceWebhook) authorized(request admissionctl.Request) admissionctl.Response {
 	var ret admissionctl.Response
+
+	// Picking OldObject or Object will suffice for most validation concerns
 	ns, err := s.renderNamespace(request)
 	if err != nil {
 		log.Error(err, "Couldn't render a Namespace from the incoming request")
 		return admissionctl.Errored(http.StatusBadRequest, err)
 	}
-	// L49-L56
-	// service accounts making requests will include their name in the group
```

Comment:
I agree that the comments here with line numbers don't make sense (and are probably just obsolete, I'm guessing from translation to golang), but I think I'd like to see the (attempts at) helpful comments themselves in this line and the one below stay, for readability